package.json

```diff
@@ -132,16 +130,12 @@
       "ts"
     ],
     "roots": [
-      "<rootDir>/eslint-rules",
       "<rootDir>/mocks",
       "<rootDir>/scripts",
-      "<rootDir>/src",
-      "node_modules/fbjs"
```

Comment:
This was relevant when we imported invariant/warning etc directly. Also the reason why tests don't currently fail in this case (but build does).